doc/source/io.rst

```diff
@@ -287,7 +287,7 @@ lineterminator : str (length 1), default ``None``
 quotechar : str (length 1)
   The character used to denote the start and end of a quoted item. Quoted items
   can include the delimiter and it will be ignored.
-quoting : int or ``csv.QUOTE_*`` instance, default ``None``
+quoting : int or ``csv.QUOTE_*`` instance, default ``0``
```

Comment:
1) The signature says `quoting=0`
2) Passing in `None` raises an error (it always has been)